### ISSUE TITLE
GarageBand Loops Wiki needs updating

### ISSUE THREAD

**Issue by RedClawx** [munki/munki#1211]
The information located on https://github.com/munki/munki/wiki/GarageBand#import-audio-content needs to be updated. Step 2 "Download the appleLoops repo" is no longer available at the location https://github.com/carlashley/appleLoops.git. A new GitHub Repo appears to have been made for downloading loop packages for GarageBand, Logic Pro, MainStage 3 at https://github.com/carlashley/loopdown

**Comment by gregneagle** [comment]
...so... update it? I have zero experience with managing GarageBand Loops.

**Comment by RedClawx** [comment]
I didn't know if I was allowed to. I'm not that familiar with the GitHub processes.

**Comment by RedClawx** [comment]
Yeah, I see the edit button. I took another quick look at loopdown. With a clean OS install the instructions may need to include other prerequisits as well, such as installing the Xcode command line tools and Python3. This is something that I would need to put on the back burner for now. I don't know when I could get to updating the information.

**Comment by gregneagle** [comment]
I'm not going to update this myself. If it's inaccurate, and no-one in the community has the time to update it, then perhaps the best course of action is for me to delete it.

**Comment by carlashley** [comment]
I've updated a small section of the wiki with my current learnings/understanding of the process.
